ee/app/livechat-enterprise/server/lib/Helper.js

```diff
@@ -173,7 +173,7 @@ export const updatePredictedVisitorAbandonment = () => {
 
 let queueTimeout;
 
-settings.get('Livechat_max_queue_wait_time', (value) => {
+settings.watch('Livechat_max_queue_wait_time', (value) => {
```

Comment:
👀  good find..

may I ask you to actually remove this watcher and use `settings.get` whenever `queueTimeout` is being used? this is an old pattern I think we should get rid of

Reply:
Ah, didn't know it was an old pattern 👀 let me change them